### PR TITLE
Hook up diffenator

### PR DESCRIPTION
**Pull request by madig**

### CODE REVIEW

File: .github/workflows/diffenator.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+          restore-keys: |
+            ${{ runner.os }}-venv-
+      - name: Build font
+        run: make build
+      - uses: f-actions/diffenator2@main
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          fetch-before: "github-release"
+          run-diffenator: true
+          path: "fonts/variable"
```
Comment (m4rc1e) [inline .github/workflows/diffenator.yaml]:
```suggestion
      - uses: f-actions/diffenator2@main
        with:
          github-token: ${{ secrets.GITHUB_TOKEN }}
          fetch-before: "github-release"
          run-diffenator: true
          path: "fonts/variable/*.ttf"
          path-before: "fonts/variable/*.ttf"
```

**Comment by m4rc1e** [comment]
I don't mind pushing this across the line if it fails again. I'd just look at the diffenate job in Google Sans, https://github.com/googlefonts/googlesans/blob/main/.github/workflows/diffenate.yml#L78-L85

File: requirements.txt [hunk 2]
```diff
@@ -170,7 +170,7 @@ gitdb==4.0.9
     # via gitpython
 gitpython==3.1.29
     # via font-v
-glyphsets==0.5.4
+glyphsets>=0.5.4
```
Comment (madig) [inline requirements.txt]: Please don't. Use `make update` to pin new deps, maybe basing your PR on https://github.com/googlefonts/googlesans-flex/pull/500.

Comment (m4rc1e) [reply requirements.txt]: @madig I'm just hacking so I actually get the results.

Comment (m4rc1e) [reply requirements.txt]: Will clean up once we get a downloadable archive with all diffs.

Comment (m4rc1e) [reply requirements.txt]: Boom. I've finally got an archive of tests, https://github.com/googlefonts/googlesans-flex/actions/runs/4617508954. This was a pita to figure out so it's pretty clear that I need better exceptions and logging.

@madig @MariannaPaszkowska are you interested in browser diffs as well? personally, I would recommend em.

Comment (m4rc1e) [reply requirements.txt]: Couldn't help myself. You'll now get images for the Thin, Regular and Black. Lmk if you want more instances.

diffbrowsers_text-Windows-10-10 0 20348-SP0_chrome_111 0 5563 111

Comment (m4rc1e) [reply requirements.txt]: @madig I've just rebased and included #500.

**Comment by m4rc1e** [comment]
@madig this is ready now. I just spoke to @chrissimpkins and @MariannaPaszkowska and for the time being we agreed that this workflow should only run when we push branches prefixed with "diff-". This matches the Google Sans implementation

**Pull request merged**